include/oneapi/dnnl/dnnl.h

```diff
@@ -436,7 +436,7 @@ dnnl_status_t DNNL_API dnnl_primitive_attr_set_scales_mask(
 ///     The set i-th bit indicates that a dedicated scaling factor is used for
 ///     each index along that dimension. Set the mask to 0 to use a common
 ///     scaling factor for the whole output tensor.
-/// @param ndims Number of group dimensions.
+/// @param group_ndims Number of group dimensions.
```

Comment:
[nitpick]
I believe `s/ndims/group_ndims/` here and below should be a separate commit within this PR.

Reply:
I already have those changes split, and I can give you a .diff; do you need it?